oxidation/libparsec/crates/types/src/time.rs

```diff
@@ -113,13 +114,69 @@ impl DateTime {
         now.into()
     }
 
-    /// Return a date-time formatted string in the rfc3339 format with a precision update to milliseconds.
-    /// Equivalent to ISO-8601
+    // We use RFC3339 format for some serialization (e.g. in `OrganizationFileLink`)
```

Comment:
Could you use doc-comment instead for each method `to_rfc3339` and `from_rfc3339` ?
That way the message would be display in IDE

Reply:
done.